t/t3510-cherry-pick-sequence.sh

```diff
@@ -93,6 +93,30 @@ test_expect_success 'cherry-pick cleans up sequencer state upon success' '
 	test_path_is_missing .git/sequencer
 '
 
+test_expect_success 'cherry-pick --skip to skip commit' '
+	pristine_detach initial &&
+	test_must_fail git cherry-pick anotherpick &&
+	test_must_fail git revert --skip &&
+	git cherry-pick --skip &&
+	test_cmp_rev initial HEAD
+'
```

Comment:
Probably also worth testing that the cherry-pick actually finished here, using `test_path_is_missing .git/sequencer`.

Reply:
AFAIK, cherry-picking single commit does not create `.git/sequencer`, how about checking for `.git/CHERRY_PICK_HEAD`?

Reply:
Right, that sounds good!